Prox/Prox/Models/Place.swift

```diff
@@ -282,6 +282,16 @@ struct OpenHours {
         return OpenHours.calendar.dateComponents(timeComponentsSet, from: dateFromTime)
     }
 
+    /** 
+    * The logic for isOpen is as follows:
```

Comment:
nit: I would prefer to use helper functions to make this logic readable in code, rather than having duplication in comments (here & inline). For example:

```
guard ... // return early if no hours today

let openingTimeToday = getDateForOpeningTime(forDate: time, hours: hours)
let closingTimeToday = ... // adds 24hrs if < openingTimeToday
let closingTimeYesterday = ... // subtracts 24hrs b/c yesterday

if let closingTimeYesterday = closingTimeYesterday,
    time < openingTimeToday {
  return time < closingTimeYesterday // We may be open overnight.
}

return time >= openingTimeToday && time < closingTimeToday
```